DependencyInjection/Configuration.php

```diff
@@ -29,6 +29,23 @@
 final class Configuration implements ConfigurationInterface
 {
     /**
+     * kernel.debug param value.
+     *
+     * @var bool
+     */
+    private $debug;
+
+    /**
+     * Configuration constructor.
```

Comment:
I would omit this description as it doesn't add any value any just keep the parameter description.